src/views/admin/donates.hbs

```diff
@@ -0,0 +1,34 @@
+<main class="main">
+  <section class="taps">
+    <input type="button" value="New Donations" class="new">
+    <input type="button" value="Current Donations" class="current">
+    <input type="button" value="Done Donations" class="done">
+  </section>
+  {{#each card}}
+  {{#unless card.state}}
+  <section class="cards--new">
+    <div class="card--taps">
+      <input type="button" value="Personal Info" class="tap--personal">
+      <input type="button" value="Donation" class="tap--donation">
+    </div>
+    {{!-- {{#if !card.state}} --}}
+    <div class="cards--info">
+      <div class="cards--info-personal">
+        <h4 class="fullName">Full Name: {{fullName}}</h4>
+        <h4 class="phoneNumber">Phone Number: {{phoneNumber}}</h4>
+        <h4 class="email">Email: {{email}}</h4>
+        <h4 class="city">City: {{cityName}}</h4>
+        {{!-- <h4 class="steet">Steet{{streetName}}:</h4> --}}
+      </div>
+      <div class="cards--info-donation hide">
+        <h4 class="piecesNumber">Number Of Pieces: {{numberOfPieces}}</h4>
+        <h4 class="gender">Quality(s): {{quality}}</h4>
+        <h4 class="quality">Gender(s): {{gender}}</h4>
+        <h4 class="season">Season(s): {{season}}</h4>
```

Comment:
@AbdallahAmmar96 Similar to my above comment, you might need `this.numberOfPieces`, `this.quality`, and so on.